pphysx3d/src/scene/mod.rs

```diff
@@ -29,24 +31,103 @@ impl PhysicsScene {
     }
 
     /// Updates the physics in the scene, such as collisions
-    pub fn update(&mut self, time_step: f32) {
+    pub fn update(&mut self) {
```

Comment:
The time step should be calculated by the user of the scene, not the scene itself imo